tests/runtest.sh

```diff
@@ -368,7 +368,7 @@ function create_core_overlay {
     if [ -d "$mscorlibDir/bin" ]; then
         cp -f -v "$mscorlibDir/bin/"* "$coreOverlayDir/" 2>/dev/null
     fi
-    cp -n -v "$testDependenciesDir"/* "$coreOverlayDir/" 2>/dev/null
+    cp -f -v "$testDependenciesDir"/* "$coreOverlayDir/" 2>/dev/null
```

Comment:
@brianrob this is actually where the xunit .dll's we want should be coming from, not "CoreClrBinDir" (That refers to the non-windows CoreCLR build). The problem was that we were using -n here, which says "Do NOT overwrite destination files" (hence the message in the logs that said we weren't overwriting those xunit .dlls)